src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -296,6 +296,10 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ngraph::Func
         {opset10::Unique::get_type_info_static(), fuse_type_to_unique_v10},
         {opset8::RandomUniform::get_type_info_static(), fuse_type_to_random_uniform_v8}};
 
+    for (const auto& it : m_additional_type_to_fuse_map) {
```

Comment:
could you update a description of ConvertPrecision transformation in convert_precision.hpp file with the info that m_additional_type_to_fuse_map allows to rewrite existing type convertors?

Reply:
Fixed. Thanks Ivan!